community/how_to_ask_on_discourse.md

```diff
@@ -1,4 +1,128 @@
 (how_to_ask_on_discourse)=
-# How to ask a question on Discourse
+# Asking Questions on the PyMC Discourse
+
+This document provides some advice for those asking questions on the [PyMC Discourse site](https://discourse.pymc.io/), the main channel for communication among the PyMC community.  If you have questions regarding modeling, PyMC/Aesara/arviz usage, or Bayesian modeling in general, please create a new post and someone is likely help you out.
+
+The items in this guide are designed to help maximize the chances that you good answers quickly.
+
+- {ref}`search`
```

Comment:
In general I'd say the toc already appears in the right sidebar automatically, but I think it is good to have it here as inline content. 

I would update the targets to be `ask_discourse/search`... or something like this to avoid potential clashes. In the case of the `search` target this is a must as it is a base sphinx protected target pointing to the search page, for the other ones it is not necessary but strongly recommended